### PR TITLE
Feature/add direct logs

### PR DESCRIPTION
**Pull request by bart88**
Implement a direct to logs link similar to the direct to terminal link on an environment.

### CODE REVIEW

**Comment by singularo** [comment]
Doesn't work with a fresh install:
```
Symfony\Component\DependencyInjection\Exception\ServiceNotFoundException: The service        [error]
"shp_custom.environment" has a dependency on a non-existent service
"plugin.manager.orchestration_provider". in
[snip]
➜  APP_ERROR: drush site-install failed.
🐳 simon@shepherd:/code$ git status
On branch feature/add-direct-logs
```

**Comment by singularo** [comment]
Should also look at disabling the caching in the view, as that makes this and the terminal link one 'feel' buggy, but all they need is a cache clear.

**Comment by singularo** [comment]
Lets also add target=_blank so we get a new tab for the terminal and logs.

**Comment by singularo** [comment]
But yeah, all works ok now, just a couple of suggestions.

File: web/modules/custom/shepherd/shp_custom/src/Service/Environment.php [hunk 1]
```diff
@@ -86,14 +95,20 @@ class Environment {
...
+                              AccountProxyInterface $currentUser,
+                              Site $site) {
     $this->requestStack = $requestStack;
     $this->entityTypeManager = $entityTypeManager;
     $this->currentRequest = $this->requestStack->getCurrentRequest();
     $this->node = $this->entityTypeManager->getStorage('node');
     $this->taxonomyTerm = $this->entityTypeManager->getStorage('taxonomy_term');
     $this->currentUser = $currentUser;
     $this->site = $site;
+    // @todo - too many cross dependancies on this service, causing install failures. Fix.
```
Comment (pingers) [inline web/modules/custom/shepherd/shp_custom/src/Service/Environment.php]: Needs refactor :+1:

File: web/modules/custom/shepherd/shp_custom/src/Service/Environment.php [hunk 2]
```diff
@@ -259,12 +275,24 @@ public function operationsLinksAlter(array &$operations, EntityInterface $entity
         'url'        => $terminal,
       ];
     }
+
+    if ($logs) {
+      $operations['logs'] = [
+        'title' => $this->t('Logs'),
+        'weight' => 4,
+        'url' => $logs,
```
Comment (pingers) [inline web/modules/custom/shepherd/shp_custom/src/Service/Environment.php]: Nitpick: indenting is different from the terminal operation above.

**Comment by pingers** [comment]
It works. :ship:

**Pull request merged**